### PR TITLE
Ecc 1007

### PR DESCRIPTION
**Pull request by vigneshwaripandi**

### CODE REVIEW

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/views/check_your_details_register.scala.html [hunk 1]
```diff
@@ -382,21 +423,56 @@
...
+            registeredAddress ++
+            dateOfEstablishment ++
+            sicCodeDisplay ++
+            summary ++
+            shortName
+            ))
+        </div>
+        @if(!isIndividual){
+        <div>
+            <h2 class="govuk-heading-m">@messages("cds.form.check-answers-vat-details")</h2>
```
Comment (NandanaRaghu) [inline app/uk/gov/hmrc/eoricommoncomponent/frontend/views/check_your_details_register.scala.html:458]: Can we check for the headers in the test?

Comment (vigneshwaripandi) [reply app/uk/gov/hmrc/eoricommoncomponent/frontend/views/check_your_details_register.scala.html:458]: The missing tests to verify the headers are  now added.

File: conf/messages.cy [hunk 2]
```diff
@@ -367,7 +367,7 @@ cds.company.name.label=Enw’r cwmni
 cds.partnership.name.label=Enw’r bartneriaeth gofrestredig
 cds.nino.label=Rhif Yswiriant Gwladol
 cds.utr.label=Rhif Cyfeirnod Unigryw y Trethdalwr (UTR)
-cds.company.utr.label=Rhif Cyfeirnod Unigryw y Trethdalwr (UTR) ar gyfer Treth Gorfforaeth
+cds.company.utr.label=TRANSLATE Corporation Tax UTR
```
Comment (darbyo) [inline conf/messages.cy:370]: Is the welsh translation not available yet?

Comment (vigneshwaripandi) [reply conf/messages.cy:370]: There is a change in the UTR label, earlier we had 'Corporation Tax UTR **number**' as suffix, but this has been removed and will be 'Corporation Tax UTR' now. Hence referred as to be TRANSLATE in future for Welsh related labels.

**Pull request merged**